src/react-components/ory/helpers/error-messages.tsx

```diff
@@ -12,13 +12,14 @@ type nodeMessageProps = {
   text: string
   id: number
   key: string
+  type: string
```

Comment:
With the new spec, we could get the types from the generated client.

Reply:
I would need to bump the client version tho which could be a bit of a refactor. Let me do that in a separate branch

Reply:
I actually already did both of those here: https://github.com/ory/elements/pull/56 and https://github.com/ory/elements/pull/57

But I needed to wait for the new client, which is released as of today. :)

Reply:
You can either take them over, if you want to, or I can do it, once I have time^^ Not sure when that is tho :D